LayoutTest/Swift/DeviceConstants.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  DeviceConstants.swift
+//  LayoutTest
+//
+//  Created by Peter Livesey on 9/21/16.
+//  Copyright © 2016 LinkedIn. All rights reserved.
+//
+
+import Foundation
+
+/**
+ This file redefines all the constants in LYTDeviceConstants for use in Swift.
+ This wasn't needed in previous version of Swift because this conversion was done automatically.
+ */
+
+public let iPhone4Width = NSNumber(value: Double(LYTiPhone4Width))
```

Comment:
I just NSNumber because it's nilable.

Setting width to 0 and setting it to nil are two different things (nil means don't change the width when running tests).

I thought about a namespace (I know it's probably going to required soon by our style guidelines), but it seems really clunky to make everyone add a bunch of `SizeConstant.` everywhere where they previously didn't need to? Especially since it's a test project.